golang.mk

```diff
@@ -28,7 +28,7 @@ OUTPUT_LINK=$(NAME)$(shell go env GOEXE)
 
 default: build
 
-vendor: go.mod go.sum
+vendor: go_generate go.mod go.sum
```

Comment:
And `go_generate` will work before vendoring?

Reply:
Yes, it **has** to be before vendoring.